### PR TITLE
A rrfs Ens3Dvar test for mpasjedi is added

### PR DESCRIPTION
**Pull request by TingLei-NOAA**
Following the similar procedure by which a rrfs test for fv3jedi was added,  a rrfs Ens3Dvar test is added for mpasjedi. 
The ctest name is  rrfs_mpasjedi_2022052619_Ens3Dvar. 
Also, now the env variable pointing to the location of the local dir containing the rrfs data is set in the machine specific module file.

### CODE REVIEW

File: rrfs-test/CMakeLists.txt [hunk 1]
```diff
@@ -47,5 +76,11 @@ ecbuild_add_test( TARGET   ${target_test}
                   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/rundir-${target_test}
                   ARGS     ${target_test}.yaml 
                   COMMAND  fv3jedi_var.x )
+set(target_test rrfs_mpasjedi_2022052619_Ens3Dvar)
```
Comment (CoryMartin-NOAA) [inline rrfs-test/CMakeLists.txt]: the build script has the option for either or both dycore right? should we add CMake to only build these tests if that dycore is built?

Comment (TingLei-NOAA) [reply rrfs-test/CMakeLists.txt]: @CoryMartin-NOAA  Good suggestion. Will add this condition for them.

**Review by ShunLiu-NOAA — DISMISSED** [review]
Looks good to me.

**Review by xyzemc — APPROVED** [review]

**Review by CoryMartin-NOAA — APPROVED** [review]
Thanks Ting!

**Comment by CoryMartin-NOAA** [comment]
@TingLei-NOAA I noticed you `force pushed`, If you do that it makes reviews harder because I can't view "changes since your last review". I know that was the old GSI way of doing things, but now we don't have to force push, we can just push each commit and squash them when merging in to develop.

**Comment by TingLei-NOAA** [comment]
@CoryMartin-NOAA Thanks for this reminder.  I will avoid using "push --force"  unless I am more strongly confident that some trivial changes are not interesting to the reviewers.

**Pull request merged**